### ISSUE TITLE
Add Support for Attribute Modifiers

### ISSUE THREAD

**Issue by SkyLicks** [funkemunky/Atlas#6]
Currently, Atlas does not have any support for the Minecraft item attribute modifiers.
https://minecraft.gamepedia.com/Attribute

Specifically, any speed modifying attribute to items would trigger false positives with any Atlas dependant anti cheat.

I think this should be added because currently, any server wity custom items is not be able to use most speed checks for any atlas dependant anti cheat.

**Comment by funkemunky** [comment]
Good idea. I can't believe I haven't responded to this one. I'll get to it for the v1.6 update. I will close this issue once it is implemented.
